assignment8.txt

```diff
@@ -0,0 +1,13 @@
+1.) fprintf() prints out to a specified output stream
+2.) printf() prints out to the standard output stream
+
+
+1.)rw opens a file for reading and writing, and doesn't overwrite whatever is in the file, it just adds to it
```

Comment:
1. “wr” -> write, then read, so the file will be cleared first for writing
2. “rw” -> read, then write, so the file will not be cleared until the first time you  output to the file
3. “w” -> overwrite the file if it already exists
4. “a” -> append (add to the end) of the file if it already exists (if not, does the same thing as write)